plugins/stack-overflow-backend/README.md

```diff
@@ -23,6 +23,7 @@ If you have a private stack overflow instance you will need to supply an API key
 stackoverflow:
```

Comment:
Could be good to call out the documentation you linked in this PR in the section above (so adopters know how to get the value)